core/src/components/fab-list/fab-list.scss

```diff
@@ -13,8 +13,8 @@
   flex-direction: column;
   align-items: center;
 
-  min-width: $fab-size;
-  min-height: $fab-size;
+  min-width: $fab-small-size + ($fab-button-small-margin * 2);
```

Comment:
These evaluate to the same value. However, the purpose of this is so the list is aligned with the parent FAB button. I decided to make this a bit more verbose so its clear where 56px is coming from. I originally thought that this was a bug since small FAB buttons can be 40px until I realized we were trying to account for the small FAB button margin too.